#!/usr/bin/env python

import roslib
roslib.load_manifest('automow_fence')

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("k_heading", double_t, 1, "Heading error gain", 10.0, 0.0, 100.0)
gen.add("k_lateral", double_t, 1, "Lateral error gain", 10.0, 0.0, 100.0)

gen.add("max_v", double_t, 1, "Maximum linear velocity (m/s)", 1.0, 0.1, 5.0)
gen.add("max_w", double_t, 1, "Maximum angular velocity (rad/s)", 0.7, 0.1, 5.0)

exit(gen.generate('automow_fence', 'automow_fence', 'Controller'))
